### PR TITLE
Add download live video frame, sub-result and consider pre-determined sandbox responses and privacy consent parameter

### PR DESCRIPTION
**Pull request by Phoebe-B**
Adding:

- download live video frame endpoint
- ability to trigger pre-determined response for Document report sub-result 
- ability to trigger pre-determined consider for multiple report checks
- privacy_notices_read_consent_given parameter

### CODE REVIEW

File: schemas/checks/check.yaml [hunk 1]
```diff
@@ -79,3 +79,12 @@
         writeOnly: true
         items:
           type: string
+      consider:
+        description: Returns a pre-determined consider sub-result in sandbox for the specific reports in the consider array.
```
Comment (superdevx64) [inline schemas/checks/check.yaml:83]: >  Returns a pre-determined consider sub-result 

I would remove "consider" from this part

**Comment by superdevx64** [comment]
I think this looks fine, thanks @Phoebe-B! My OpenAPI is a bit rusty though so @ciaran16 may have some comments

**Review by superdevx64 — APPROVED** [review]

File: paths/live_videos.yaml [hunk 2]
```diff
@@ -62,3 +62,26 @@
...
+      responses:
+        "200":
+          description: The live video frame's binary data.
+          content:
+            "*/*":
+              schema:
+                type: string
+                format: binary
+        default:
+          $ref: ../responses/default_error.yaml
```
Comment (ciaran16) [inline paths/live_videos.yaml:87]: Probably a good idea to configure your IDE/editor to automatically insert a newline at the end of files

**Review by ciaran16 — COMMENTED** [review]
Looks good 👍  have you tried validating or generating something from this? It tends to be very easy to make a mistake that's difficult to spot with OpenAPI

**Comment by superdevx64** [comment]
@Phoebe-B With hindsight, it'd be good to add `privacy_notices_read_consent_given` to this as well given that that's now active in v2 and v3

File: schemas/checks/check.yaml [hunk 3]
```diff
@@ -47,6 +47,9 @@
       applicant_id:
         type: string
         description: The ID of the applicant to do the check on.
+      privacy_notices_read_consent_given:
```
Comment (ciaran16) [inline schemas/checks/check.yaml:50]: I think this needs `writeOnly: true`, as in API v3 we're only adding to the request object

Comment (Phoebe-B) [reply schemas/checks/check.yaml:50]: Done!

**Review by ciaran16 — APPROVED** [review]

**Pull request merged**